liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/BasicSupport.java

```diff
@@ -193,6 +199,15 @@ protected void init() throws MojoExecutionException, MojoFailureException {
                 installDirectory = checkServerHome(assemblyArchive);
                 log.info(MessageFormat.format(messages.getString("info.variable.set"), "installDirectory", installDirectory));
             } else if (assemblyArtifact != null) {
+                // check for liberty.runtime.version property which overrides any version set in the assemblyArtifact
+                if (libertyRuntimeVersion != null) {
+                    if (assemblyArtifact.getVersion() != null) {
+                        log.warn("The assemblyArtifact version " + assemblyArtifact.getVersion() + " is overwritten by the liberty.runtime.version value "+ libertyRuntimeVersion +".");
```

Comment:
I think we should log as `info` here instead of `warn`, since this behavior is intentional.